data/pathfinder/paizo/roleplaying_game/core_rulebook/cr_equipmods.lst

```diff
@@ -216,47 +216,47 @@ Wield Three Step Greater No Penalty	KEY:Special Quality ~ Wield Size / 3 Steps G
+Brilliant Energy				KEY:Special Ability ~ Brilliant Energy ~ Weapon	FORMATCAT:FRONT	NAMEOPT:NORMAL	TYPE:Ammunition.Weapon		PLUS:4	VISIBLE:QUALIFY																																								SOURCEPAGE:p.470				PREMULT:2,[PRETYPE:1,Enhancement,WeaponEnhancement],[PRETYPE:1,Weapon,Ammunition]																																																					SPROP:ignores armor and shield bonuses including enhancement, cannot harm undead, constructs, and objects
+Dancing					KEY:Special Ability ~ Dancing ~ Melee		FORMATCAT:FRONT	NAMEOPT:NORMAL	TYPE:Weapon.Melee			PLUS:4	VISIBLE:QUALIFY																																								SOURCEPAGE:p.346															PRETYPE:3,WeaponEnhancement,Weapon,Melee																																																						SPROP:can be loosed to attack on its own																																																																																																									ASSIGNTOALL:YES
+Defending					KEY:Special Ability ~ Defending ~ Melee		FORMATCAT:FRONT	NAMEOPT:NORMAL	TYPE:Weapon.Melee			PLUS:1	VISIBLE:QUALIFY																																								SOURCEPAGE:p.347															PRETYPE:3,WeaponEnhancement,Weapon,Melee																																																						SPROP:can transfer some or all of its enhancement bonus to AC
+Disruption					KEY:Special Ability ~ Disruption ~ Weapon		FORMATCAT:FRONT	NAMEOPT:NORMAL	TYPE:Weapon.Bludgeoning		PLUS:2	VISIBLE:QUALIFY																																								SOURCEPAGE:p.347															PRETYPE:3,WeaponEnhancement,Weapon,Bludgeoning																																																					SPROP:undead hit must make Will save (DC 14) or be destroyed
+Distance					KEY:Special Ability ~ Distance ~ Ranged		FORMATCAT:FRONT	NAMEOPT:NORMAL	TYPE:Weapon.Ranged		PLUS:1	VISIBLE:QUALIFY																																								SOURCEPAGE:p.347															PRETYPE:3,WeaponEnhancement,Weapon,Ranged														BONUS:EQMWEAPON|RANGEMULT|2																																				SPROP:range increment doubled																																																																																																											ASSIGNTOALL:YES
+Flaming					KEY:Special Ability ~ Flaming ~ Weapon		FORMATCAT:FRONT	NAMEOPT:NORMAL	TYPE:Ammunition.Weapon		PLUS:1	VISIBLE:QUALIFY																																								SOURCEPAGE:p.347							PREMULT:2,[PRETYPE:1,Enhancement,WeaponEnhancement],[PRETYPE:1,Weapon,Ammunition]																																																					SPROP:+1d6 fire damage
```

Comment:
This should just be WeaponEnhancement. Enhancement shouldn't exist.

Reply:
Enhancement exists because it can be an Ammunition also

Reply:
Ammo is a subgroup of Weapons. It should not be separated.

Reply:
PREMULT:2,[PRETYPE:1,Enhancement,WeaponEnhancement], <--- still exists